src/server.ts

```diff
@@ -1,7 +1,12 @@
+import path from "path"
+import express from "express"
 import app from '../api';
 
 const PORT = process.env.PORT || 3333
 
+app.use(express.static(path.join(__dirname,'public')))
```

Comment:
Define middleware in `api/index.ts`, because Vercel get the Application object in this file.